src/components/Masthead/Masthead.tsx

```diff
@@ -61,23 +61,28 @@ export const Masthead = ({ className }: MastheadProps) => {
         <Logo
           isLink
           className={cnb('su-w-[17rem] sm:su-w-[24rem] su-transition-all', isAtTop ? 'lg:su-w-[32rem]' : 'lg:su-w-[28rem]')}
-          color={!isAtTop && isScrollingBack ? 'black' : 'white'}
         />
         {/* The scale3d here solves a Firefox only rendering bug with blurry curved borders when using transform */}
-        <FlexBox alignItems="center" className={cnb('su-transition-all', isAtTop ? '' : 'lg:[transform:scale3d(0.75,0.75,0.75)]')}>
+        <FlexBox alignItems="center" className={cnb('su-transition-all', isAtTop ? '' : 'su-origin-right lg:[transform:scale3d(0.75,0.75,0.75)]')}>
```

Comment:
Added transform origin to start on the right hand side, so that the right edge of the masthead buttons align with the centered container when it's scaled down on scrolling up.